style: Inconsistent macro names changed

rruuaanng opened the pull request:
Fix incorrect header file pre-macro names in 'include/zephyr/xen', 'include/zephyr/display' and 'include/zephyr/mgmt'.

good
https://github.com/zephyrproject-rtos/zephyr/blob/4a8a035bd7b9178e6ae31d56e931aa9b9a40f9b9/include/zephyr/display/ssd16xx.h#L8

https://github.com/zephyrproject-rtos/zephyr/blob/4a8a035bd7b9178e6ae31d56e931aa9b9a40f9b9/include/zephyr/mgmt/updatehub.h#L14

bad
https://github.com/zephyrproject-rtos/zephyr/blob/4a8a035bd7b9178e6ae31d56e931aa9b9a40f9b9/include/zephyr/mgmt/osdp.h#L12

https://github.com/zephyrproject-rtos/zephyr/blob/4a8a035bd7b9178e6ae31d56e931aa9b9a40f9b9/include/zephyr/display/cfb.h#L12

stephanosio commented on include/zephyr/xen/public/xen.h:
```diff
@@ -26,8 +26,8 @@
 
-#ifndef __XEN_PUBLIC_XEN_H__
-#define __XEN_PUBLIC_XEN_H__
+#ifndef ZEPHYR_INCLUDE_XEN_PUBLIC_XEN_H_
+#define ZEPHYR_INCLUDE_XEN_PUBLIC_XEN_H_
```
[These are Xen headers were copied from Xen repository](https://github.com/zephyrproject-rtos/zephyr/commit/bfe9989d9f95a8b3ae2308698d24c2d76767d390).

We do not want to create unnecessary diffs like these for external headers.

rruuaanng replied:
Oh, I see. I will roll back all changes to xen.

rruuaanng replied:
But I have a question, that is, can I change the header files in the non-public directory under the xen directory?

rruuaanng committed:
style: Inconsistent macro names changed

Fix incorrect header file pre-macro names in
'include/zephyr/app_memory', 'include/zephyr/display'
and 'include/zephyr/mgmt'.

Signed-off-by: James Roy <rruuaanng@outlook.com>

rruuaanng committed:
style: Inconsistent macro names changed

Fix incorrect header file pre-macro names in
'include/zephyr/shell' and 'include/zephyr/task_wdt'.

Signed-off-by: James Roy <rruuaanng@outlook.com>
